wrapper/virt_v2v_wrapper.py

```diff
@@ -487,6 +487,21 @@ def main():
                                                   host.get_uid(),
                                                   host.get_gid())
 
+        if 'luks_keys_vault' not in data:
+            data['luks_keys_vault'] = '/tmp/v2v_luks_keys_vault.json'
```

Comment:
Do we need a default? If yes, it may be worth using some other directory then `/tmp`, e.g. `/etc` or user's home directory. What do you think? Also what do you think about adding a check and refusing to load file that is readable/writable by someone else than owner?

Reply:
Good points. Changed to use user's home directory and added checks on permissions.